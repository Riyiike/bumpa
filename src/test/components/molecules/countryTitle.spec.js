import React from "react";
import { shallow } from 'enzyme'
import CountryTitle from "../../../components/molecules/countryTitle";




describe('test mainPage', () => {
    it('test mainPage', () => {
        const btn = shallow(<CountryTitle />)
        expect(btn).toMatchSnapshot();
    })
})