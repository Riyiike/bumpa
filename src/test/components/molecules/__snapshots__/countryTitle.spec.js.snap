// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test mainPage test mainPage 1`] = `
<h2
  className="font-bold text-lg sm:text-2xl mb-4 lg:mb-5 sm:col-span-2 flex items-center gap-2 sm:gap-3"
/>
`;
